### PR TITLE
changed 'filter-by language' instead of '--filter-by-language'

### PR DESCRIPTION
**Pull request by Dumidu921**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: coalib/parsing/DefaultArgParser.py [hunk 1]
```diff
@@ -184,7 +184,7 @@ def default_arg_parser(formatter_class=None):
         help='list all bears')
 
     outputs_group.add_argument(
-        '-l', '--filter-by-language', nargs='+', metavar='LANG',
+        '-l', 'filter-by language', nargs='+', metavar='LANG',
```
Comment (Nosferatul) [inline coalib/parsing/DefaultArgParser.py]: here should be `--filter-by-language` that's the argument that you write on the console. don't change it

File: coalib/parsing/DefaultArgParser.py [hunk 2]
```diff
@@ -39,7 +39,7 @@ def default_arg_parser(formatter_class=None):
 To find out what kind of analysis coala offers for the languages you use, visit
 http://coala.io/languages, or run::
 
-    $ coala --show-bears --filter-by-language C Python
+    $ coala --show-bears filter-by language C Python
```
Comment (Nosferatul) [inline coalib/parsing/DefaultArgParser.py]: here I think it should be `.... filter-by C Python`

**Comment by damngamerz** [comment]
Also please look at our commit [guidelines](coala.io/commit).

File: coalib/parsing/DefaultArgParser.py (hunk 2 above)
Comment (Dumidu921) [inline coalib/parsing/DefaultArgParser.py]: @Nosferatul  is it ok with filter-by C python ? without 'language'

Comment (Nosferatul) [inline coalib/parsing/DefaultArgParser.py]: well, the language are `C Python`

**Comment by Nosferatul** [comment]
and you have some unwanted commits, please rebase properly. Take a look http://api.coala.io/en/latest/Developers/Git_Basics.html#rebasing

**Comment by Dumidu921** [comment]
Please help me to pass the above checks.

**Comment by aashraybhandar1** [comment]
Please check the commit guidelines at https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html

**Comment by aashraybhandar1** [comment]
Please check the commit guidelines at https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html

**Comment by jayvdb** [comment]
Duplicate of https://github.com/coala/coala/pull/4709

**Pull request closed without merging**